lib/assert.js

```diff
@@ -38,6 +38,7 @@ class AssertionError extends Error {
 		this.improperUsage = opts.improperUsage || false;
 		this.operator = opts.operator;
 		this.values = opts.values || [];
+		this.stackTraceLimit = Infinity;
```

Comment:
Is this sufficient? I don't see a reference to `stackTraceLimit` as an *instance* property in https://github.com/v8/v8/wiki/Stack-Trace-API.

Instead I suspect we need to manually call `Error.captureStackTrace(this)` below, if `opts.stack` is falsy.

Reply:
You are right. Thank you for pointing this out.
I made change to the code.
Thanks